drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java

```diff
@@ -597,6 +605,26 @@ void onUpdatedLockStatusEvent( final @Observes UpdatedLockStatusEvent event ) {
         }
     }
 
+    void onIssueSelectedEvent( final @Observes IssueSelectedEvent event ) {
+        if ( event == null ) {
```

Comment:
It probably cannot be null (if invoked by CDI).. I guess I got carried away with defensive programming (this time!)